app/views/projects/_clients.haml

```diff
@@ -13,7 +13,7 @@
           - @clients.each do |service|
             %tr
               %td
-                - if ! @project.confidential? || can_edit_projects? || ! project.confidential_for_user?(user)
+                - if ! @project.confidential? || can_edit_projects? || ! @project.confidential_for_user?(user)
```

Comment:
style without the space after the ! operator is more conventional ([ref](https://rubystyle.guide/#no-space-bang))
```suggestion
                - if !@project.confidential? || can_edit_projects? || !@project.confidential_for_user?(user)
```